test/knex.test.js

```diff
@@ -259,6 +260,77 @@ describe('transaction', function () {
 
     throw new Error('expected the call to throw')
   })
+
+
+  it('adopt-commit', async () => {
+
+    let foo1_id
+
+    const trx = await knex(DbConfigPG).transaction()
+    const txseneca  = await si.entity.adopt({handle:trx})
+
```

Comment:
verify that txseneca has a transaction
use: https://github.com/senecajs/seneca-entity/blob/master/src/entity.ts#L135

Reply:
Hmmm, did something like this, correct me with I'm wrong:

```
const isTransaction = !!txseneca.entity.state().transaction
expect(isTransaction).equal(true)
```

Reply:
yes